cmd/browser_pools.go

```diff
@@ -0,0 +1,683 @@
+	Short: "Acquire a browser from the pool",
+	Args:  cobra.ExactArgs(1),
+	RunE:  runBrowserPoolsAcquire,
+}
+
+var browserPoolsReleaseCmd = &cobra.Command{
+	Use:   "release <id-or-name>",
+	Short: "Release a browser back to the pool",
+	Args:  cobra.ExactArgs(1),
+	RunE:  runBrowserPoolsRelease,
+}
+
+var browserPoolsFlushCmd = &cobra.Command{
+	Use:   "flush <id-or-name>",
+	Short: "Flush idle browsers from the pool",
+	Args:  cobra.ExactArgs(1),
+	RunE:  runBrowserPoolsFlush,
+}
+
+func init() {
+	// list flags
+	browserPoolsListCmd.Flags().StringP("output", "o", "", "Output format: json for raw API response")
+
+	// create flags
+	browserPoolsCreateCmd.Flags().String("name", "", "Optional unique name for the pool")
+	browserPoolsCreateCmd.Flags().Int64("size", 0, "Number of browsers in the pool")
+	_ = browserPoolsCreateCmd.MarkFlagRequired("size")
+	browserPoolsCreateCmd.Flags().Int64("fill-rate", 0, "Fill rate per minute")
+	browserPoolsCreateCmd.Flags().Int64("timeout", 0, "Idle timeout in seconds")
+	browserPoolsCreateCmd.Flags().Bool("stealth", false, "Enable stealth mode")
+	browserPoolsCreateCmd.Flags().Bool("headless", false, "Enable headless mode")
+	browserPoolsCreateCmd.Flags().Bool("kiosk", false, "Enable kiosk mode")
+	browserPoolsCreateCmd.Flags().String("profile-id", "", "Profile ID")
+	browserPoolsCreateCmd.Flags().String("profile-name", "", "Profile name")
+	browserPoolsCreateCmd.Flags().Bool("save-changes", false, "Save changes to profile")
+	browserPoolsCreateCmd.Flags().String("proxy-id", "", "Proxy ID")
+	browserPoolsCreateCmd.Flags().StringSlice("extension", []string{}, "Extension IDs or names")
+	browserPoolsCreateCmd.Flags().String("viewport", "", "Viewport size (e.g. 1280x800)")
+
+	// get flags
+	browserPoolsGetCmd.Flags().StringP("output", "o", "", "Output format: json for raw API response")
+
+	// update flags
+	browserPoolsUpdateCmd.Flags().String("name", "", "Update the pool name")
+	browserPoolsUpdateCmd.Flags().Int64("size", 0, "Number of browsers in the pool")
+	browserPoolsUpdateCmd.Flags().Int64("fill-rate", 0, "Fill rate per minute")
+	browserPoolsUpdateCmd.Flags().Int64("timeout", 0, "Idle timeout in seconds")
+	browserPoolsUpdateCmd.Flags().Bool("stealth", false, "Enable stealth mode")
+	browserPoolsUpdateCmd.Flags().Bool("headless", false, "Enable headless mode")
+	browserPoolsUpdateCmd.Flags().Bool("kiosk", false, "Enable kiosk mode")
+	browserPoolsUpdateCmd.Flags().String("profile-id", "", "Profile ID")
+	browserPoolsUpdateCmd.Flags().String("profile-name", "", "Profile name")
+	browserPoolsUpdateCmd.Flags().Bool("save-changes", false, "Save changes to profile")
+	browserPoolsUpdateCmd.Flags().String("proxy-id", "", "Proxy ID")
+	browserPoolsUpdateCmd.Flags().StringSlice("extension", []string{}, "Extension IDs or names")
+	browserPoolsUpdateCmd.Flags().String("viewport", "", "Viewport size (e.g. 1280x800)")
+	browserPoolsUpdateCmd.Flags().Bool("discard-all-idle", true, "Discard all idle browsers")
```

Comment:
should default be false? seems like a potential footgun

Reply:
yeah i could go either way on this so maybe we keep it as is (default true) and see how it goes